build-deb.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Build a Jellyfin .deb file with Docker on Linux
+# Places the output .deb file in the parent directory
+
+set -o xtrace
+set -o errexit
+set -o pipefail
+set -o nounset
+
+date="$( date +%s )"
+curdir="$( pwd )"
+tmpdir="$( mktemp -d )"
+curuser="$( whoami )"
+
+docker build ${curdir} --tag jellyfin-debuild-${date} --file ${curdir}/Dockerfile.debian_package
```

Comment:
Nah, I can understand what original line did (because `--tag` is not ambiguous but `-t` could be anything and requires me to know Docker CLI).
So I vote for long options in scripts where you write once and read multiple times.